apps/system/js/lock_screen_window_manager.js

```diff
@@ -398,21 +388,13 @@
    * @this {LockScreenWindowManager}
    * @memberof LockScreenWindowManager
    */
-  LockScreenWindowManager.prototype.initWindow =
-    function lwm_initWindow() {
-      var req = window.SettingsListener.getSettingsLock()
-        .get('lockscreen.enabled');
-      req.onsuccess = () => {
-        this.states.ready = true;
-        if (true === req.result['lockscreen.enabled'] ||
-           'true' === req.result['lockscreen.enabled']) {
-          this.states.enabled = true;
-        } else if (false === req.result['lockscreen.enabled'] ||
-                   'false' === req.result['lockscreen.enabled']) {
-          this.states.enabled = false;
-        }
-        this.openApp();
-      };
+  LockScreenWindowManager.prototype['_observe_lockscreen.enabled'] =
```

Comment:
I would suggest you at least to make these special (obvious different) 'methods' as:

    // Comment the reason why these code should with different conventions.
    // Maybe a simple 'BaseModule methods'.
    LockScreenWindowManager.prototype._observe_lockscreen = {
        enabled () {}....
    }

Since these two conventions unfortunately need to exist at the same time, it should be explicitly distinguished from the ordinary methods. I think the reason why we hope to have only one convention in one app is still valid, so if the current style is still valid and you have no plan (at least I've heard none of them) to change all existing methods to the 'new' style, we shouldn't adopt such similar but with totally different intention ways to name a 'method'.